src/Components/CreateImageWizard/utilities/useValidation.tsx

```diff
@@ -262,14 +262,10 @@ export function useHostnameValidation(): StepValidation {
 
 export function useKernelValidation(): StepValidation {
   const kernel = useAppSelector(selectKernel);
+  const errors = {};
 
   if (!isKernelNameValid(kernel.name)) {
```

Comment:
personally, I prefer this approach 
```
  const kerneler = !isKernelNameValid(kernel.name) ? 'Invalid format.' : '';

  return {
    errors: {
      kernel: kerneler,
    },
    disabledNext: !!kerneler,
  };
```
it is more readable for me :) WDYT?
I checked that and it is working the same

Reply:
Sure, I've updated the code. :+1:  Will open a follow up for the other validators that use `Object.assign()`

Reply:
I lied, I've added the changes to this PR after all :zipper_mouth_face: